README.md

```diff
@@ -121,28 +57,19 @@ import { getAuthenticatorFromEnvironment } from 'ibm-cloud-sdk-core';
 const iamAuthenticator = getAuthenticatorFromEnvironment('my-service');
 ```
 
-### CloudPakForData
-Used to manage and authenticate with tokens for services on IBM Cloud Pak for Data.
-
-Programmatic:
-```js
-import { CloudPakForDataAuthenticator } from 'ibm-cloud-sdk-core';
```

Comment:
Maybe add a line like "For more details about authenticators, see <link to AUTHENTICATION.md>".  You can see what I did in the go core's README.md file.